src/MessagingInput/MessagingInput.tsx

```diff
@@ -21,47 +21,63 @@ interface Props {
   onBlur?: () => void;
 }
 
-export const MessagingInput: React.FC<Props & { ref?: React.Ref<TextArea> }> = React.forwardRef(
-  (props, ref) => {
-    const { className, recipientName, value, onChange, onSubmit, onBlur } = props;
+export interface MessagingInputHandle {
+  focus(): void;
+}
+
+const MessagingInputRenderFunction: React.ForwardRefRenderFunction<MessagingInputHandle, Props> = (
```

Comment:
Some context on the change here: before we were exposing a `ref` to `TextArea`, which I thought felt a bit to specific (it shouldn't matter which component this uses under the hood). Instead, I only expose the `focus` method on the `MessagingInputHandle` interface and use `useImperativeHandle` to implement that.